client/my-sites/email/controller.js

```diff
@@ -190,20 +191,19 @@ export default {
 		next();
 	},
 
+	// Steps to reproduce it: https://github.com/Automattic/wp-calypso/pull/87098#email-management-titan-set-up-thank-you
```

Comment:
Similarly, I would avoid pointing to a PR. If we are looking to provide context of this feature, it's best to describe the scenario rather than pointing to the PR and refer to test steps.

```suggestion
	// Thank you page for when user sets up a mailbox after purchasing a Titan subscription.
```

After going through the flow again, I think it should be changed altogether. The Congrats Page is meant to be shown in the "checkout experience" (without sidebar, without the admin bar). We can still proceed with as-is in this PR, so that the old Congrats page is removed. But I would suggest creating a followup task to:

* Redirect users back to the Email Management page after setting up a mailbox (no payment was involved, the action was really part of management, it doesn't make sense for us to show the Congrats page)
* Display a notice "Your email is now ready to use!" (copy taken from old Congrats page).

Reply:
👍 Created https://github.com/orgs/Automattic/projects/865/views/1?pane=issue&itemId=52788499

Reply:
Thanks for creating the issue! The description doesn't look exactly accurate though, I'll clarify in the issue itself.